### ISSUE TITLE
Implicit relative imports are only supported in Python 2

### ISSUE THREAD

**Issue by fservantwiremind** [astrikos/wetransfer-python-sdk#4]
https://chrisyeh96.github.io/2017/08/08/definitive-guide-python-imports.html
Therefore, statements like:
```python
from logger import LOGGER
```
lead to `ImportError` when using Python 3, that can be troublesome for many people

**Comment by astrikos** [comment]
Yeah I am pretty sure that not's the only thing. I haven't listed on purpose python3 support in the `setup.py` classifiers https://github.com/astrikos/wetransfer-python-sdk/blob/master/setup.py#L35-L44
It's on my TODO list though, I am also not happy about it.
